css/components/button.css

```diff
@@ -95,19 +88,16 @@
 .btn-icon-item{
     height: max-content;
 } 
-
+ 
 /* floating button */
-
-
 .btn-float{
     margin: 0.2rem;
     min-width: 2rem;
     min-height: 2rem;
```

Comment:
since its the floating button, you can set a specific width/height instead of using the min properties

Reply:
okay i think you are right

Reply:
done width and height